src/hovers/decorationState.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  DecorationOptions,
+  ExtensionContext,
+  MarkdownString,
+  Position,
+  Range,
+  window,
+  workspace,
+} from "vscode";
+import { Hover } from "../binary/requests/hovers";
+import { getLogoPath } from "../consts";
+
+const decorationType = window.createTextEditorDecorationType({
+  after: { margin: "0 0 0 1rem" },
+});
+
+let decoration: DecorationOptions | null;
+let decorationsDebounce: NodeJS.Timeout;
```

Comment:
the types are incorrect. Both are `undefined`, but both are not defined as undefined. Are you using vscode to lint the typescript?